### PR TITLE
App version in Navigation Drawer

### LINKED ISSUE #291: View app version in navigation menu
**Describe the issue**

Viewing the app version in app is important while reporting bugs and fixing issues. Similar to PSLab Android app, display the current app version in navigation bar

**Would you like to work on the issue?**

Issue for GSoC

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #291 

**Changes**: Added App version in Navigation Drawer.

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3284093/app-debug.zip)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -72,4 +72,5 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="bit24_pref_key">bit24</string>
     <string name="advanced_mode_pref_key">advancedMode</string>
     <string name="axis_desc_time">Time</string>
+    <string name="app_version">1.0.0</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: This is wrong. Check pslab-android source code to see how it's done. Also please once again, don't add to end of resource files.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/values/strings.xml]: Okay.

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 2]
```diff
@@ -31,7 +31,7 @@
         </menu>
     </item>
 
-    <item android:title="Communicate">
+    <item android:title="Other">
```
Comment (suresh-gandhi) [inline app/src/main/res/menu/activity_main_drawer.xml]: @jddeep dont hardcode

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 3]
```diff
@@ -41,6 +41,10 @@
                 android:id="@+id/nav_send"
                 android:icon="@drawable/ic_menu_send"
                 android:title="Send" />
```
Comment (suresh-gandhi) [inline app/src/main/res/menu/activity_main_drawer.xml]: here also and even in share

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/values/strings.xml]: Done.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
https://github.com/fossasia/pslab-android/blob/development/app/build.gradle#L19 put this line in gradle file and refer it as @strings/version in navigation drawer layout. Just 4-5 lines of code. Tell me if you still can't do. I'll send a PR. Takes only 5 minutes.

**Comment by jddeep** [comment]
> https://github.com/fossasia/pslab-android/blob/development/app/build.gradle#L19 put this line in gradle file and refer it as @strings/version in navigation drawer layout. Just 4-5 lines of code. Tell me if you still can't do. I'll send a PR. Takes only 5 minutes.

Ah! New thing learnt. Thanks.

**Comment by jddeep** [comment]
Screenshot from 2019-06-13 09-14-11
@CloudyPadmal What's up with Travis CI build? `ANDROID_HOME` path is set fine here.

**Comment by adityastic** [comment]
@jddeep travis has its own Environment variables, travis variables have no relation to local Environment variables

File: app/src/main/java/io/neurolab/main/MainActivity.java [hunk 4]
```diff
@@ -26,6 +28,8 @@
     private CardView focusButton;
     private CardView relaxButton;
     private CardView memGraphButton;
+    private View headerView;
+    private TextView versionTextView;
```
Comment (adityastic) [inline app/src/main/java/io/neurolab/main/MainActivity.java]: Don't define them here. Declare directly inside onCreate

**Comment by jddeep** [comment]
> 
> 
> @jddeep travis has its own Environment variables, travis variables have no relation to local Environment variables

Yeah I found it. Was researching.Trying to fix the Travis build issue.

File: app/src/main/res/layout/nav_header_main.xml [hunk 5]
```diff
@@ -28,4 +28,13 @@
         android:textColor="@android:color/white"
         android:textSize="@dimen/text_size_large" />
 
+    <TextView
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/nav_header_main.xml]: These are not needed anymore

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/nav_header_main.xml (hunk 5 above)
Comment (jddeep) [reply app/src/main/res/layout/nav_header_main.xml]: Done

**Review by CloudyPadmal — APPROVED** [review]

**Review by suresh-gandhi — APPROVED** [review]

**Pull request merged**